Gemfile

```diff
@@ -7,3 +7,9 @@
 source 'https://rubygems.org'
 
 gem 'stove'
+gem 'test-kitchen'
```

Comment:
You shouldn't need this in the Gemfile if you're using DK for testing

Reply:
but if youre not...

Reply:
why wold you not?